easybuild/tools/systemtools.py

```diff
@@ -219,30 +222,40 @@ def get_cpu_speed():
         try:
```

Comment:
The affinity mask is a POSIX thing right? So why not try the BSD thing and then fall back to the affinity mask?